desktop/libs/notebook/src/notebook/connectors/jdbc.py

```diff
@@ -50,9 +50,15 @@ def decorator(*args, **kwargs):
         else:
           raise QueryError, _('%s: is the DB Proxy server running?') % message, sys.exc_info()[2]
       elif 'Access denied' in message:
-        raise AuthenticationRequired, '', sys.exc_info()[2]
+        if sys.version_info[0] > 2:
+          raise AuthenticationRequired('').with_traceback(sys.exc_info()[2])
+        else:
+          raise AuthenticationRequired, '', sys.exc_info()[2]
```

Comment:
Nit: Let's only support for Py3. No need for the if condition because soon we'll be removing all Py2 related stuff from Hue.

```python
  else:
    raise QueryError, _('%s: is the DB Proxy server running?') % message, sys.exc_info()[2]
elif 'Access denied' in message:
  raise AuthenticationRequired('').with_traceback(sys.exc_info()[2])
```
 should be good.

Reply:
Hi @Harshg999 . So, I should make this change at all the required places or for this change only

Reply:
For this only